### PR TITLE
MINOR: add config docs to manage-topics

### PR DESCRIPTION
**Pull request by vvcephei**
The config documentation from https://github.com/apache/kafka/pull/3459/files#diff-8dfad184a2604987442db10d8bebf8bc got lost at some point. This PR adds it back.

### CODE REVIEW

**Comment by vvcephei** [comment]
rendered:
1.0:
10-docs

1.1:
11-docs

2.0:
20-docs

2.1:
21-docs

**Comment by vvcephei** [comment]
@JimGalasyn @guozhangwang @mjsax,

Please review this PR to correct a docs omission.

Thanks!
-John

**Comment by vvcephei** [comment]
Updated:
2.1:
21-docs

2.0:
20-docs

1.1:
11-docs

1.0:
10-docs

File: 10/streams/developer-guide/manage-topics.html [hunk 1]
```diff
@@ -67,10 +67,38 @@
...
+      <p>
+        Internal topics are used internally by the Kafka Streams application
+        while executing, for example the changelog topics for state stores.
+        These topics are created by the application and are only used by that
+        stream application.
+      </p>
+      <p>
+        If security is enabled on the Kafka brokers, you must grant the
+        underlying clients admin permissions so that they can
+        create internal topics set. For more information, see
```
Comment (JimGalasyn) [inline 10/streams/developer-guide/manage-topics.html:79]:
```suggestion
        create the internal topics set. For more information, see
```

File: 11/streams/developer-guide/manage-topics.html [hunk 2]
```diff
@@ -67,10 +67,38 @@
...
+      <p>
+        Internal topics are used internally by the Kafka Streams application
+        while executing, for example the changelog topics for state stores.
+        These topics are created by the application and are only used by that
+        stream application.
+      </p>
+      <p>
+        If security is enabled on the Kafka brokers, you must grant the
+        underlying clients admin permissions so that they can
+        create internal topics set. For more information, see
```
Comment (JimGalasyn) [inline 11/streams/developer-guide/manage-topics.html:79]:
```suggestion
        create the internal topics set. For more information, see
```

File: 20/streams/developer-guide/manage-topics.html [hunk 3]
```diff
@@ -67,10 +67,38 @@
...
+      <p>
+        Internal topics are used internally by the Kafka Streams application
+        while executing, for example the changelog topics for state stores.
+        These topics are created by the application and are only used by that
+        stream application.
+      </p>
+      <p>
+        If security is enabled on the Kafka brokers, you must grant the
+        underlying clients admin permissions so that they can
+        create internal topics set. For more information, see
```
Comment (JimGalasyn) [inline 20/streams/developer-guide/manage-topics.html:79]:
```suggestion
        create the internal topics set. For more information, see
```

File: 21/streams/developer-guide/manage-topics.html [hunk 4]
```diff
@@ -67,10 +67,38 @@
...
+      <p>
+        Internal topics are used internally by the Kafka Streams application
+        while executing, for example the changelog topics for state stores.
+        These topics are created by the application and are only used by that
+        stream application.
+      </p>
+      <p>
+        If security is enabled on the Kafka brokers, you must grant the
+        underlying clients admin permissions so that they can
+        create internal topics set. For more information, see
```
Comment (JimGalasyn) [inline 21/streams/developer-guide/manage-topics.html:79]:
```suggestion
        create the internal topics set. For more information, see
```

**Review by JimGalasyn — APPROVED** [review]
LGTM, with a couple of suggestions.

**Review by guozhangwang — APPROVED** [review]
@vvcephei do you have a PR in `kafka` repo so that we would have the content in 2.2+ releases as well?

**Pull request merged**